### ISSUE TITLE
Reading the values

### ISSUE THREAD

**Issue by skinatro** [avanishsubbiah/material-you-theme#47]
Is there any way to read these values like you can read and use pywal values?

**Comment by ardjuh** [comment]
Yes you can get the color mapping from ~/.config/gtk-4.0/gtk.css

**Comment by Dani9OO** [comment]
#56 I made a PR that runs pywal on background change, you might want to take a look 😃

**Comment by ardjuh** [comment]
Can you maybe look at the comments of the PR?
